cpp/src/lib/rora_proxy_client.h

```diff
@@ -85,6 +85,10 @@ class RoraProxy_client : public Proxy_client {
                  const std::vector<std::shared_ptr<sequences::Update>> &,
                  std::vector<proxy_protocol::object_info> &);
 
+  virtual void apply_sequence(const std::string &namespace_,
+                              const write_barrier, const sequences::Sequence &,
+                              std::vector<proxy_protocol::object_info> &);
+
```

Comment:
There should be no need for this / the implementation in `rora_proxy_client.cc` - the method in `generic_proxy_client` (which need not be `virtual`) should work fine?